src/Connections/GithubRepo.ts

```diff
@@ -415,8 +425,8 @@ export class GitHubRepoConnection extends CommandConnection implements IConnecti
         return this.state.priority || super.priority;
     }
 
-    public async onStateUpdate(stateEv: MatrixEvent<unknown>) {
-        this.state = stateEv.content as GitHubRepoConnectionState;
+    protected validateConnectionState(content: unknown) {
+        return content as GitHubRepoConnectionState;
```

Comment:
Seems dangerous, can't we use the `validateState` function?

Reply:
We could, as long as there was no strong reason why it wasn't being done already.

Reply:
6b08b7b